workflow/personal_genome_input_wf.cwl

```diff
@@ -0,0 +1,80 @@
+cwlVersion: v1.2
+class: Workflow
+id: personal-genome-input-wf
+label: Personal Genome Input Workflow
+doc: "Filter an input VCF and remove annotations"
+
+inputs:
+  # Strip, subset and PASS vars
+  input_vcf: {type: File, secondaryFiles: ['.tbi']}
+  strip_info: {type: 'string?', doc: "If given, remove previous annotation information based on INFO file, i.e. to strip VEP info, use INFO/ANN"}
+  output_basename: string
```

Comment:
What's the point of asking for this input? It's not actually used for any of the saved outputs.